content/en/metrics/overview.md

```diff
@@ -0,0 +1,146 @@
+   - [Custom checks][5], which are used to collect metrics from custom applications or unique systems. You define the logic for the check yourself in the Agent's configuration files. See [Writing a Custom Agent Check][6] for more information.
+   - [DogStatsD][7], a metrics aggregation service bundled with the Datadog Agent. DogStatsD implements the [StatsD][8] protocol with some Datadog-specific extensions.
+   - [Marketplace integrations][9] installed on the Agent. The [Datadog Marketplace][10] is a digital marketplace where Technology Partners can list their paid offerings to Datadog users.
+{{% /collapse-content %}}
+
+{{% collapse-content title="Cloud integrations" level="h4" %}}
+Also known as authentication-based integrations, these integrations are set up in Datadog. You provide the credentials for obtaining metrics, and Datadog makes API calls on your behalf to collect them. Common examples are cloud provider integrations, Slack, and PagerDuty.
+{{% /collapse-content %}} 
+
+{{% collapse-content title="Datadog API" level="h4" %}}
+You can send metrics directly to the [Metrics API][11].
+{{% /collapse-content %}} 
+
+{{% collapse-content title="Metrics from other Datadog Services" level="h4" %}}
+Some Datadog services, such as Synthetic Testing, provide standard metrics by default. Other services can be configured to generate custom metrics. This section covers the available options and provides links to relevant documentation.
+
+##### Logs
+
+Log-based metrics allow you to generate custom metrics from any logs ingested into Datadog. See [Generate Metrics from Ingested Logs][12] for more information.
+
+##### APM
+
+- [Trace metrics][13] are standard metrics that capture request counts, error counts, and latency measures on your applications instrumented with Datadog APM. You can also [generate custom metrics from your ingested spans][15].
+- [Runtime metrics][14] are standard metrics that provide additional insights into an application’s performance.
+
+##### Real User Monitoring (RUM)
+
+- [Web and mobile vitals][16] are standard metrics that provide performance scores for browser and mobile applications, respectively.
+- [Custom metrics from RUM events][17] enable you to summarize the data from your RUM events. 
+
+##### Processes
+
+[Process-based metrics][19] are custom metrics you can generate in the [Processes page][25].
+
+##### Events
+
+[Event-based metrics][20] are custom metrics you can generate in the [settings page of Event Management][26].
+
+##### Synthetic Monitoring & Continuous Testing Metrics
+
+[Synthetic Monitoring & Continuous Testing Metrics][18] are standard metrics that are generated to help track test duration, latency, and more.
+
+##### Account Management
```

Comment:
`synthetics.*` metrics aren't listed as custom metrics in the Plan & Usage page, and I've additionally located some internal resources that verify this. Let me know if there is any additional context around this I'm unaware of, or if you have any feedback.

Reply:
This might sound a bit confusing, but this section only pertains to product where its possible to generate custom metrics (thus billable), which includes: logs, APM, rum and processes. Can we please fix up this section accordingly?

Reply:
Thanks for clarifying, I updated this section in [aaedfa1](https://github.com/DataDog/documentation/pull/25830/commits/aaedfa1dd0509ef6c265a3c89d308ae6faa5dcc5) to:
- Remove anything about standard metrics
- Emphasize the use cases for creating the different custom metrics

Reply:
I see that this still has products which aren't logs, APM, rum and processes. @buraizu any chance we can remove this?